app/views/publishers/organisations/_vacancy_awaiting_feedback.html.slim

```diff
@@ -1,5 +1,7 @@
-= form_for vacancy, url: organisation_job_statistics_path(vacancy.id), html: { id: vacancy.id, method: "patch", class: "vacancy-feedback__form", data: { option_not_selected_message: t("messages.jobs.feedback.inline_error") } }
-  .card-component id=dom_id(vacancy)
+= form_for vacancy_statistics_form(vacancy), url: organisation_job_statistics_path(vacancy.id), html: { id: vacancy.id, method: "patch" } do |f|
```

Comment:
the html attributes can just be given without the `html` hash

Reply:
Turns out this is not true